src/Api/NavigationUtils.ts

```diff
@@ -263,7 +263,13 @@ export const findStaticNavigatorByStateKey = (
   state: NavigationState,
   stateKey: string,
 ): StaticTreeNavigator => {
-  const routeName = getRouteNameByStateKey(state, stateKey)
+  let routeName: string
+  try {
+    routeName = getRouteNameByStateKey(state, stateKey)
+  } catch (e) {
+    log.debug('Missing route name for state key, using root navigator', { stateKey, state })
+    routeName = ROOT_NAVIGATOR_ID
+  }
```

Comment:
probably it should be covered differently, if key non empty and doesn't exist then probably it's an issue, but if key is undefined for root navigator as we were expecting, then we should return ROOT_NAVIGATOR_ID

Reply:
After checking and running the code, I found out that `stateKey` is always string, so I'm bringing back the old code. If I encounter the crash I will fix accordingly.